src/Avalonia.Base/Platform/PlatformColorValues.cs

```diff
@@ -31,7 +31,7 @@ public record PlatformColorValues
     /// <summary>
     /// System theme variant or mode.
     /// </summary>
-    public PlatformThemeVariant ThemeVariant { get; init; }
+    public PlatformThemeVariant? ThemeVariant { get; init; }
```

Comment:
Ideally, this property shouldn't be nullable. OS should report an actual theme variant at any point, regardless of the app state. In fact, it shouldn't rely on Application.Current either.

Reply:
Done